connectors/connectors.sh

```diff
@@ -50,13 +54,32 @@ get_connector_url() {
 
   # spark-bigquery
   if [[ $name == spark-bigquery ]]; then
-    # DATAPROC_VERSION is an environment variable set on the cluster.
-    # We will use this to determine the appropriate connector to use
+    readonly -A LT_DP_VERSION_SCALA=([1.5]="2.11" # On Dataproc images < 1.5, use Scala 2.11
+                                     [2.0]="2.12" # On Dataproc images < 2.0, use Scala 2.12
+                                     [2.1]="2.12" # On Dataproc images < 2.1, use Scala 2.12
+                                     [2.2]="2.13" # On Dataproc images < 2.2, use Scala 2.13
+                                    )
+    # DATAPROC_IMAGE_VERSION is built from an environment variable set on the
+    # cluster.  We will use this to determine the appropriate connector to use
     # based on the scala version.
-    if [[ $(min_version "$DATAPROC_VERSION" 1.5) == 1.5 ]]; then
-      local -r scala_version=2.12
-    else
-      local -r scala_version=2.11
+
+    for LT_DP_VER in 1.5 2.0 2.1 2.2 ; do
+      if ! test -v scala_version && compare_versions_lt $DATAPROC_IMAGE_VERSION ${LT_DP_VER} ; then
```

Comment:
nit: prefer `"${var}"` over `$var`. This is the convention of Dataproc shell script.

Reply:
I'll take care of it